src/test/java/tutoraid/logic/parser/ArgumentTokenizerTest.java

```diff
@@ -49,8 +50,7 @@ private void assertArgumentPresent(ArgumentMultimap argMultimap, Prefix prefix,
     }
 
     private void assertArgumentAbsent(ArgumentMultimap argMultimap, Prefix prefix) {
-        // If the prefix isn't present, the string ("") will be returned
-        assertTrue(argMultimap.getValue(prefix).isPresent());
+        assertFalse(argMultimap.getValue(prefix).isPresent());
```

Comment:
Was this the one you mentioned not to do also?

Reply:
I'm reverting a change that shouldn't have been made with this